CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## Unreleased
+
+###
+- Cloud only: Support session cookies.
```

Comment:
This is not necessarily cloud only. It'd potentially apply to any load balancer (NginX?) that can be configured to return the appropriate set-cookie header

Reply:
OK I'll change that.